### PR TITLE
fix Koding Issue #9142

### PR DESCRIPTION
**Pull request by lionheart1022**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by lionheart1022** [comment]
I fixed the issue.

**Comment by lionheart1022** [comment]
This is the screencast.
[https://www.dropbox.com/s/p7uoba9atf1f3wd/Rec%2010-28-16%201.mp4?dl=0](url)

File: client/app/lib/defaultshortcuts/editor.json [hunk 1]
```diff
@@ -523,7 +523,7 @@
         "ctrl+y"
       ],
       [
-        "command+y"
+        "command+shift+z"
```
Comment (hakankaradis) [inline client/app/lib/defaultshortcuts/editor.json]: 👍

File: client/ide/lib/workspace/panes/ideeditorpane.coffee [hunk 2]
```diff
@@ -83,7 +83,7 @@ module.exports = class IDEEditorPane extends IDEPane
     { ace } = @aceView
 
     ace.ready =>
-      @getEditor().setValue content, 1
+      @getEditor().session.setValue content, 1
```
Comment (hakankaradis) [inline client/ide/lib/workspace/panes/ideeditorpane.coffee]: 👍

**Review by hakankaradis — COMMENTED** [review]
LGTM

**Comment by usirin** [comment]
@lionheart1022 Thank you for your submission, can you please provide detailed instructions as what this is solving? Just writing down the issue number to title is not enough.

Thanks.

**Comment by lionheart1022** [comment]
Issue number is #9142.

-Go to Stack-Editor section by editing a current stack or create new one
-Go to Stack template, custom variables or read me tabs
-Press Command (Ctrl) + Z
-You will see empty stack template
-or write something then press Command + Z. it will continue to redo changes till see the template empty.

- client/app/lib/defaultshortcuts/editor.json
The shortcut key of redo was Command(Ctrl) + Y, rather than Command(Ctrl)+Shift+Z.
Hakan told that redo key didn't work, so I updated it to Command+Shift+Z from Command+Y.

-client/ide/lib/workspace/panes/ideeditorpane.coffee

I synchronized ACE editor and session - that means I set the content as initial content of ACE editor.
I'd like you to review it asap.
Thanks.

**Comment by lionheart1022** [comment]
If you press the undo key just after opening stack-editor page, the content will be cleared even you don't change anything.
What I had to do is to fix that issue - the initial content should be there even the user press undo key continuously.
And the second issue was to make the redo key enable.
Now undo(command+z) and redo(command+shift+z) work well.

Please let me know if you more questions.
Thanks

**Comment by usirin** [comment]
LGTM, will merge after a deploy.

**Review by usirin — APPROVED** [review]

**Comment by usirin** [comment]
Good Job @lionheart1022

**Comment by lionheart1022** [comment]
Thanks

**Pull request merged**